src/components/Sidebar.vue

```diff
@@ -27,61 +34,124 @@ export default {
   },
   methods: {
     showNewList(list) {
-      this.$emit('list', list.component);
+      this.$emit("list", list.component);
     }
   }
 };
 </script>
 
 <style lang="scss" scoped>
-@import "../scss/variables/colors";
+@import '../scss/variables/colors';
+@import '../scss/variables/sizes';
 
 .sidebar-container {
-  background: $sidebar-blue;
+  background-color: $sidebar-blue;
   display: flex;
   flex-direction: column;
   height: 100vh;
   justify-content: space-between;
   max-width: 300px;
-  padding: 90px 20px 10px;
   position: sticky;
   top: 0;
   width: 100%;
 }
 
 .sidebar-header {
-  height: 180px;
+  display: flex;
+  flex-direction: column;
+  padding: 90px 20px 0 30px;
 }
 
 .content-links {
   align-items: flex-start;
+  color: $white;
   display: flex;
   flex-direction: column;
-  color: $white;
+  padding: 0 20px 0 30px;
 }
 
 .simple-link {
+  color: $science-blue;
   font-size: 16px;
   line-height: 19px;
+  position: relative;
   text-transform: uppercase;
-  color: $white;
+  transition: all $transition-duration $transition-function;
 
   &:not(:last-child) {
     margin-bottom: 20px;
   }
+
+  &::before {
+    background-color: $white;
+    content: '';
+    height: 100%;
```

Comment:
do we have a `full-height` class?

Reply:
It isn't possible because is a `::before`